app/components/NoticeFormat.tsx

```diff
@@ -61,11 +63,26 @@ export function NoticeFormat({
         </>
       ),
     },
+    {
+      value: 'json' as ValueType,
+      label: 'JSON',
+      description: (
+        <>
+          Preliminary new notice types available only via JSON (other notice
+          types under development).
+        </>
+      ),
+    },
   ]
 
   return (
     <>
-      <input type="hidden" name={name} value={currentValue} />
+      <input
+        type="hidden"
+        name={name}
+        value={currentValue}
+        onChange={(e) => (stateFunction ? stateFunction(e) : null)}
```

Comment:
Wouldn't this result in multiple invocation of the state function? Since the value of the hidden field only every changes programmatically, when we set it.

```suggestion
```

Reply:
Good catch, who knows what past-Dakota was thinking. 

Cleaning this up in the next commit